app/src/main/kotlin/com/wire/android/ui/home/newconversation/contacts/ContactsScreen.kt

```diff
@@ -56,25 +59,31 @@ fun ContactsScreenContent(
             Modifier
                 .fillMaxSize()
         ) {
-            LazyColumn(
-                state = lazyListState,
-                modifier = Modifier.weight(1f),
-            ) {
-                folderWithElements(
-                    header = { stringResource(R.string.label_contacts) },
-                    items = state.contacts
-                ) { contact ->
-                    ContactItem(
-                        name = contact.name,
-                        userStatus = contact.userStatus,
-                        belongsToGroup = newGroupContacts.contains(contact),
-                        addToGroup = { addContactToGroup(contact) },
-                        removeFromGroup = { removeContactFromGroup(contact) }
-                    )
+            if (isLoading) {
+                Box(Modifier.fillMaxSize()) {
+                    CircularProgressIndicator(Modifier.align(Alignment.Center))
+                }
```

Comment:
Just an idea, this might seem simple right now but in case we are using the same `CircularProgressIndicator`, could we make this a CustomView or something related so we don't duplicate this specific code, and if in the future the loading design is changed we won't need to change everywhere in the codebase.

Let me know what you think about it

Reply:
true, agree on it :) will update the PR later on